tests/test_fluent_session.py

```diff
@@ -202,7 +200,7 @@ def _freeze_fluent(s):
     alive = timeout_loop(
         get_container,
         5.0,
-        args=(session.connection_properties.cortex_host,),
+        args=(session.cortex_host,),
```

Comment:
Is this going to work? I think this was a mistaken search and replace, the root session object should not have details about the `cortex_host`